pkg/gadgets/trace/dns/tracer/tracer.go

```diff
@@ -44,14 +44,40 @@ func NewTracer() (*Tracer, error) {
 		return nil, fmt.Errorf("failed to load asset: %w", err)
 	}
 
+	latencyCalc, err := newDNSLatencyCalculator()
```

Comment:
The same LRU cache is used for matching packets from different container / different network namespaces. I am concerned of the interaction it could have. I will run some tests.